setup

```diff
@@ -151,6 +151,11 @@ install() {
         EXTRA_FLAG="--use-host-config"
     fi
 
+    # SecurityLabelNested not supported so far in CentOS Stream 9 or lower
```

Comment:
Is it due to bootc?

Reply:
No, me and Leo found this testing CentOS @Yarboa